CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# [VuFind Developer Manual](https://vufind.org/wiki/development)
+
+## Prerequisites
+
+ - [Community Code of Conduct](https://github.com/vufind-org/vufind/CODE_OF_CONDUCT.md)
+ - [Coding Standards](https://vufind.org/wiki/development:coding_standards) - How to conform to VuFind's style.
```

Comment:
On this page, the template section says "guide forthcoming." We should address that somehow. If nothing else, we should note that we're now using php-cs-fixer to auto-format templates, so that enforces basic styles. It may be enough to simply acknowledge that fact.